src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs

```diff
@@ -241,8 +242,8 @@ internal bool EnableAutoPurge
                         } else {
                             throw new ArgumentException(CouchbaseLiteErrorMessage.InvalidHeartbeatInterval);
                         }
-                    } else {
-                        this[HeartbeatIntervalKey] = 0;
+                    } else { // Backward compatible if null is set
+                        this[HeartbeatIntervalKey] = Constants.DefaultReplicatorHeartbeat.Ticks / TimeSpan.TicksPerSecond;
```

Comment:
Constants.DefaultReplicatorHeartbeat.TotalSeconds ?

Reply:
I remember the reason we chose ticks/ticks per second is because we want to long value result instead of double result returned from TotalSeconds...

Reply:
Interesting, I have completely forgotten this reason.  Do you remember what bad stuff happened?